x-pack/plugins/infra/public/pages/logs/page_template.tsx

```diff
@@ -20,5 +22,24 @@ export const LogsPageTemplate: React.FC<LazyObservabilityPageTemplateProps> = (
     },
   } = useKibanaContextForPlugin();
 
-  return <PageTemplate {...pageTemplateProps} />;
+  const { sourceStatus } = useLogSourceContext();
+  const noDataConfig: KibanaPageTemplateProps['noDataConfig'] =
+    sourceStatus?.logIndexStatus !== 'missing'
+      ? undefined
+      : {
+          solution: 'Observability',
```

Comment:
**Logs**

I'm not sure if checking for `!missing` is correct here

Reply:
I think this is exactly right, actually. The values are here: 

https://github.com/elastic/kibana/blob/9b456ac0fe706528a686060c8c013b9998e7c45e/x-pack/plugins/infra/common/http_api/log_sources/get_log_source_status.ts#L41-L45

"empty" means there is just no data right now, but "missing" implies the indices haven't even been set up.

@weltenwort / @Kerry350 confirm for me on this?

Reply:
Yep, that's my understanding 👍

Reply:
Thanks for confirming @Kerry350 !